Return to Office/Building Access Appointments.md

```diff
@@ -0,0 +1,300 @@
+# Buildings Endpoints
+
+## General Setup
+
+All examples are using https://github.com/axios/axios#axios-api.
+```js
+axios = require('axios')
+PROOF_API_HOST = process.env.PROOF_API_HOST
+PROOF_API_TOKEN = process.env.PROOF_API_TOKEN
+```
+
+## POST: Creating a Building Access Appointment
+
+This endpoint creates a building access appointment (appointment) in the current user's government (based on the API token).
+
+**Currently not supported**
+<details>
+  <summary>will look something like this</summary>
+
+
+```js
+axios({
+  method: 'POST',
+  url: `${PROOF_API_HOST}/api/building_access_appointments`,
+  headers: {
+    'Accept': 'application/json',
+    'Authorization': `Bearer ${PROOF_API_TOKEN}`,
+    'Content-Type': 'application/json',
+  },
+  data: {
+    "building_access_appointment": {
+      'building_id': '<integer not null>',
+      'floor': '<string/integer? not null>',
+      'end_at': '<datetime not null>',
+      'start_at': '<datetime not null>',
+      'user_id': '<integer not null>',
+      'recurring_appointment_profile_id': '<integer>',
+      'routing_id': '<integer',
+    }
+  }
```

Comment:
Existing docs (i.e. in this repo) give all examples via cURL ...

Reply:
Andrew, Danella and I opted to switch to JS + Axios as it is much more readable and seems to be the "lingua franca" of the web (for this rev of the API docs).